releasenotes/notes/ironic-expand-check_cipher_suite_errors-c8915956be17a28c.yaml

```diff
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Bumped Ironic tag to fix a bug where some cipher sute errors for
```

Comment:
If I was being super picky I'd say that this line should explicitly mention _container image_ tags

Reply:
Done